tests_requre/service/test_api.py

```diff
@@ -182,3 +182,61 @@ def test_get_testing_farm_results(
     assert response_dict[1]["status"] == "new"
 
     assert response_dict[2]["target"] == SampleValues.chroots[1]
+
+
+# Test Get Projects
+def test_get_projects_list(client, clean_before_and_after, a_copr_build_for_pr):
+    response = client.get(url_for("api.projects_projects_list"))
+    response_dict = response.json
+    assert len(response_dict) == 1
+    assert response_dict[0]["namespace"] == SampleValues.repo_namespace
+    assert response_dict[0]["repo_name"] == SampleValues.repo_name
+    assert response_dict[0]["project_url"] == SampleValues.project_url
+    assert response_dict[0]["prs_handled"] == 1
+
+
+# Test Get Project's Pull Requests
+def test_get_projects_prs(client, clean_before_and_after, a_copr_build_for_pr):
+    response = client.get(
+        url_for(
+            "api.projects_projects_p_rs",
+            forge="github.com",
+            namespace="the-namespace",
+            repo_name="the-repo-name",
+        )
+    )
+    response_dict = response.json
+    assert len(response_dict) == 1
+    assert response_dict[0]["pr_id"] is not None
+    assert response_dict[0]["builds"][0]["build_id"] == SampleValues.build_id
+    assert response_dict[0]["builds"][0]["status"] == "pending"
+
+
+# Test Get Project's Issues
+def test_get_projects_issues(client, clean_before_and_after, an_issue_model):
+    response = client.get(
+        url_for(
+            "api.projects_project_issues",
+            forge="github.com",
+            namespace="the-namespace",
+            repo_name="the-repo-name",
+        )
+    )
+    response_dict = response.json
+    assert response_dict["issues"][0] == SampleValues.issue_id
+
+
+# Test Get Project's Releases
```

Comment:
Could we have this the python way and make them doc-string? :)

Iirc, this is how Go makes comments on functions, but in Python this should really be a doc-string.